### PR TITLE
Cargo department is once again brown in the crew manifest

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
Back in #60578, the crew manifest's hardcoded jobs were replaced. For most of the crew manifest this meant no change, but the crew manifest CSS assumed that cargo would be called supply, meaning that it wouldn't be coloured. This fixes that.

<details>

Before
image

After
image
</details>

## Why It's Good For The Game
Fixes an oversight

## Changelog

:cl:
fix: the cargo department is once again coloured in the crew manifest
/:cl:

### CODE REVIEW

**Comment by msgerbs** [comment]
Would be nice if there was some consistency. The radio channel calls it supply, as does the security officer assignment, the ban system, and probably other stuff I don't know enough to find.

**Comment by Fikou** [comment]
i like supply more than cargo

**Review by Fikou — CHANGES_REQUESTED** [review]
i would prefer we instead change the name to supply, keeping it consistent with most stuff calling them supply

File: tgui/packages/tgui/styles/interfaces/CrewManifest.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ $department_map: (
   'Medical': colors.$teal,
   'Misc': colors.$white,
   'Science': colors.$purple,
-  'Supply': colors.$brown,
+  'Cargo': colors.$brown,
```
Comment (Fikou) [inline tgui/packages/tgui/styles/interfaces/CrewManifest.scss:10]: you are forsaking the alphabetical order btw

Comment (dragomagol) [reply tgui/packages/tgui/styles/interfaces/CrewManifest.scss:10]: This is a grievous crime that I have committed

**Comment by dragomagol** [comment]
> Would be nice if there was some consistency. The radio channel calls it supply, as does the security officer assignment, the ban system, and probably other stuff I don't know enough to find.

#define JOB_SECURITY_OFFICER_SUPPLY "Security Officer (Cargo)"

**Comment by msgerbs** [comment]
> > Would be nice if there was some consistency. The radio channel calls it supply, as does the security officer assignment, the ban system, and probably other stuff I don't know enough to find.
> 
> #define JOB_SECURITY_OFFICER_SUPPLY "Security Officer (Cargo)"
> 
> #define DEPARTMENT_CARGO "Cargo"
> 
> There really is no consistency. We could make a decision on where we use which, or if we want all one or the other. But there's a ton of references to the department as Cargo.

What's this?

`#define SEC_DEPT_SUPPLY "Supply"`

`#define EXP_TYPE_SUPPLY "Supply"`

**Comment by JohnFulpWillard** [comment]
The PR they linked has changed much of 'Supply' to 'Cargo' and seems to have been done intentionally. If you want to revert that, you can do it separately, but I don't see why a Fix PR for the crew manifest's colors should be expected to revert the intentional (?) work of another PR.

**Comment by msgerbs** [comment]
> The PR they linked has changed much of 'Supply' to 'Cargo' and seems to have been done intentionally. If you want to revert that, you can do it separately, but I don't see why a Fix PR for the crew manifest's colors should be expected to revert the intentional (?) work of another PR.

I'm not suggesting that Supply is the correct name or that it should be changed. Merely pointing out other places it's still called Supply to emphasize the inconsistency.

**Pull request merged**